spec.html

```diff
@@ -24392,20 +24392,27 @@ <h1>
       <emu-alg>
         1. Let _accessObj_ be OrdinaryObjectCreate(%Object.prototype%).
         1. If _kind_ is ~field~, ~method~, ~accessor~, or ~getter~, then
-          1. Let _getterClosure_ be a new Abstract Closure with no parameters that captures _name_ and performs the following steps when called:
-            1. Let _o_ be the *this* value.
-            1. If _name_ is a String, return ? Get(_o_, _name_).
-            1. Else, return ? PrivateGet(_name_, _o_).
-          1. Let _getter_ be CreateBuiltinFunction(_getterClosure_, 0, *""*, &laquo; &raquo;).
+          1. Let _getterClosure_ be a new Abstract Closure with parameter (_obj_) that captures _name_ and performs the following steps when called:
+            1. If _obj_ is not an Object, throw a *TypeError* exception.
+            1. If _name_ is either a String or a Symbol, return ? Get(_obj_, _name_).
+            1. Else, return ? PrivateGet(_name_, _obj_).
+          1. Let _getter_ be CreateBuiltinFunction(_getterClosure_, 1, *""*, &laquo; &raquo;).
           1. Perform ! CreateDataPropertyOrThrow(_accessObj_, *"get"*, _getter_).
         1. If _kind_ is ~field~, ~accessor~, or ~setter~, then
-          1. Let _setterClosure_ be a new Abstract Closure with parameters (_value_) that captures _name_ and performs the following steps when called:
-            1. Let _o_ be the *this* value.
-            1. If _name_ is a String, perform ? PrivateSet(_name_, _o_, _value_).
-            1. Else, perform ? Set(_o_, _name_, _value_, *true*).
+          1. Let _setterClosure_ be a new Abstract Closure with parameters (_obj_, _value_) that captures _name_ and performs the following steps when called:
+            1. If _obj_ is not an Object, throw a *TypeError* exception.
+            1. If _name_ is either a String or a Symbol, perform ? PrivateSet(_name_, _obj_, _value_).
+            1. Else, perform ? Set(_obj_, _name_, _value_, *true*).
```

Comment:
Are `PrivateSet` and `Set` swapped (also before this PR)?

Reply:
They are. I'll fix this, thanks.